site/e2e/helpers.ts

```diff
@@ -43,6 +44,15 @@ export function requiresEnterpriseLicense() {
   test.skip(!enterpriseLicense);
 }
 
+// requiresTerraform by default is enabled.
+export function requiresTerraform() {
+  if (requireTerraformTests) {
+    return;
+  }
+
+  test.skip(!requireTerraformTests);
```

Comment:
The early return is unnecessary/redundant here

```suggestion
  test.skip(!requireTerraformTests);
```